### DISCUSSION TITLE
The Best speech analysis and synthesis for multilingual online Voice conversion applications

### DISCUSSION THREAD

**Discussion by mahdeslami11** [community/community#44641] [Programming Help] [answered]
In the link below, there is a valuable list of articles related to speech synthesis: https://github.com/wenet-e2e/speech-synthesis-paper
In your opinion, what is the best method of speech analysis and synthesis for the use of multilingual online Voice conversion system? Witch methods have you used so far to implement the analyzer and synthesizer in the Voice conversion system? Please explain about the strengths and weaknesses of the method you use, if any.

**Comment by Maryammsv** [comment]
Two basic methods of speech synthesis are considered: 1) the generation of speech from stored segments, and 2) the generation of speech through continuous control of the various speech parameters individually; in the latter case, the parameters may be physiological or acoustical.
Linear prediction has become an important method for speech analysis and synthesis. Although the basic mathematical foundations of the theory of linear prediction were established in the early 1940s, the method did not find widespread use for speech analysis and synthesis until the mid-1960s. Most speech analysis until then was based exclusively on frequency-domain techniques. Since then, the linear prediction has become the predominant method of speech processing. 
The Concatenative speech synthesis technique is a corpus-based technique that uses some pre-recorded speech samples (words, syllables, half-syllables, phonemes, di- phones or triphones) in a database and produces the output speech by concatent- ing appropriate units based on the entered text utterances

**Comment by sanagolmarzi** [comment]
In the name of God
In my opinion, all methods have many advantages and disadvantages, but in general, it is impossible to say which one is the best because it depends on the structure and methods we are considering.

**Comment by alirezaahmadiii** [comment, marked as answer]
A method for speech analysis and synthesis comprising steps of sampling a short-period power spectrum of an input speech with a basic frequency, applying a cosine polynomial model to thus obtained sample points to determine the spectrum envelope, calculating the mel cepstrum coefficients from said spectrum envelope, and effecting speech synthesis utilizing said mel cepstrum coefficients as the filter coefficients of a mel logarithmic spectrum approximation filter.2. A method according to claim 1, wherein said mel cepstrum coefficients are calculated by converting the frequency axis of the spectrum envelope into a mel approximation scale and applying an inverse FFT operation to the mel logarithmic spectrum envelope.3. A method according to claim 1, wherein said mel cepstrum coefficients are calculated by applying an inverse FFT process to the spectrum envelope to determine the cepstrum coefficients and applying regressive equations on said cepstrum coefficients.4. A method according to claim 3, wherein said regressive equations consists of following equations:Cα(m) = µ(m0), m = 0, 1, 2, ...5. A method or apparatus for analysing and synthesising speech, in which the spectrum envelope of speech is determined by sampling a power spectrum and fitting a curve to the sample points, cepstrum coefficients are calculated from the spectrum envelope, and speech is synthesised using the calculated cepstrum coefficients. And also, I used random CNN in the project, but in the proposal part and presenting a new method and suggesting the improvement of the project I used Acoustic Model(Tacotron V1 : https://arxiv.org/abs/1703.10135 )This is a many-to-one voice conversion system. The main significance of this work is that we could generate a target speaker's utterances without parallel data like <source's wav, target's wav>, <wav, text> or <wav, phone>, but only waveforms of the target speaker. (To make these parallel datasets needs a lot of effort.) All we need in this project is a number of waveforms of the target speaker's utterances and only a small set of <wav, phone> pairs from a number of anonymous speakers. This project has the best results among all current works.
But And the cost is:
1.	Heavy architecture. the architecture is to training 2 networks, Net1 classifier and Net2 synthesizer  and combine them together
2.	Delicate dataset.
3.	Not general. For example The model was trained only for Kate Winslet's voice transfer. If we want to transfer to Obama's voice, we need to gather Obama's voice data and train that network again.

**Comment by maissa10336** [comment]
Abstract
This paper presents an analysis of speech synthesis quality achieved by simultaneously performing voice conversion and language code-switching using multilingual VQ-VAE speech synthesis in German, French, English and Italian. In this paper, we utilize VQ code indices representing phone information from VQ-VAE to perform code-switching and a VQ speaker code to perform voice conversion in a single system with a neural vocoder. Our analysis examines several aspects of code-switching including the number of language switches and the number of words involved in each switch. We found that speech synthesis quality degrades after increasing the number of language switches within an utterance and decreasing the number of words. We also found some evidence of accent transfer when performing voice conversion across languages as observed when a speaker's original language differs from the language of a synthetic target utterance. We present results from our listening tests and discuss the inherent difficulties of assessing accent transfer in speech synthesis. Our work highlights some of the limitations and strengths of using a semi-supervised end-to-end system like VQ-VAE for handling multilingual synthesis. Our work provides insight into why multilingual speech synthesis is challenging and we suggest some directions for expanding work in this area.
   Deep Feature Transfer Learning for Automatic Pronunciation Assessment

**Comment by Shabnamlatifian** [comment]
the best metod which i have found
Modeling voices for multiple speakers and multiple languages with one [speech synthesis system](https://www.sciencedirect.com/topics/computer-science/speech-synthesis-system) has been a challenge for a long time, especially in low-resource cases. This paper presents two approaches to achieve cross-lingual multi-speaker text-to-speech (TTS) and code-switching synthesis under two training scenarios: (1) cross-lingual synthesis with sufficient data, (2) cross-lingual synthesis with limited data per speaker. Accordingly, a novel [TTS synthesis](https://www.sciencedirect.com/topics/computer-science/speech-synthesis) model and a non-autoregressive multi-speaker voice conversion model are proposed. The TTS model designed for sufficient-data cases has a Tacotron-based structure that uses shared phonemic representations associated with numeric language ID codes. As for the data-limited scenario, we adopt a framework cascading several speech modules to achieve our goal. In particular, we proposed a non-autoregressive many-to-many voice conversion module to address multi-speaker synthesis for data-insufficient cases. Experimental results on speaker similarity show that our proposed voice conversion module can maintain the voice characteristics well in data-limited cases. Both approaches use limited bilingual data and demonstrate impressive performance in cross-lingual synthesis, which can deliver fluent foreign speech and even code-switching speech for monolingual speakers.

**Comment by mahdigodarzi** [comment]
i have used this metod
This paper presents an analysis of speech synthesis quality achieved by simultaneously performing voice conversion and language code-switching using multilingual VQ-VAE speech synthesis in German, French, English and Italian. In this paper, we utilize VQ code indices representing phone information from VQ-VAE to perform code-switching and a VQ speaker code to perform voice conversion in a single system with a neural vocoder. Our analysis examines several aspects of code-switching including the number of language switches and the number of words involved in each switch. We found that speech synthesis quality degrades after increasing the number of language switches within an utterance and decreasing the number of words. We also found some evidence of accent transfer when performing voice conversion across languages as observed when a speaker's original language differs from the language of a synthetic target utterance. We present results from our listening tests and discuss the inherent difficulties of assessing accent transfer in speech synthesis. Our work highlights some of the limitations and strengths of using a semi-supervised end-to-end system like VQ-VAE for handling multilingual synthesis. Our work provides insight into why multilingual speech synthesis is challenging and we suggest some directions for expanding work in this area.

**Comment by SepidehKianasab** [comment]
Speech recognition software processes speech uttered in a natural language and converts it into readable text with a high degree of accuracy, using artificial intelligence (AI), machine learning (ML), and natural language (NLP) techniques. This article discusses the key features of speech recognition software and the top 10 tools in this segment. 
Speech recognition software is defined as a technology that can process speech uttered in a natural language and convert it into readable text with a high degree of accuracy, using artificial intelligence (AI), machine learning (ML), and natural language (NLP) techniques.

**Comment by GeronimoKTT** [comment]
@mahdeslami11 I suggest you try [www.nemesyslabs.com](https://www.nemesyslabs.com/)
